src/components/FirstGameRow/FirstGameRow.spec.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import FirstGameRow from "./FirstGameRow";
+import {mount} from 'enzyme';
+import {GameState} from "../../services/CommonTypes";
+import StarsDisplay from "../StarsDisplay/StarsDisplay";
+import SelectedNumbersDisplay from "../SelectedNumbersDisplay/SelectedNumbersDisplay";
+
+describe('FirstGameRow component tests', () => {
+    it("should render correctly when game is not initialized", () => {
+        const expectedNumberOfStarsForUninitializedGame = 0
+        const roundNumber = 2
+        const numberOfStars = 5
+        const selectedNumbers = [1, 2, 3]
+        const unselectNumber = jest.fn()
+        const wrapper = mount(<FirstGameRow roundNumber={roundNumber} numberOfStars={numberOfStars}
```

Comment:
I think this component needs to be split into more components, each concerned with a single task.

Reply:
This would also reduce the amount of individual test cases and improve coverage